### DISCUSSION TITLE
Info on TF binding predictions or motif data

### DISCUSSION THREAD

**Discussion by danish2562022** [netZoo/netZooPy#311] [Q&A]
I want to construct Gene Regulatory network for the brain tissue using PANDA, and I have RNA-Seq data, and PPI from STRING DB. I do not have TF binding predictions or motif data, and I am completely new to this field. Could you please let me know from where I can collect/download TF binding predictions or motif data? Or do we need to derive from RNA-Seq Data using some algorithms ?

**Comment by violafanfani** [comment]
Hi @danish2562022,
sorry for the late reply. 

For the motif input you need a tf-gene matrix that constitutes the prior the regulatory network. These networks are based on Transcription Factor Binding Sites (TFBS) detected in the promoter regions of target genes. For most of our applications we download the Homo sapiens transcription factor motifs from CIS-BP (http://cisbp.ccbr.utoronto.ca/) and we map them to the promoter regions of the Ensembl genes using [FIMO](https://meme-suite.org/meme/doc/fimo.html).


I highly recommend you check out the "Building a regulation prior network for netZoo tools" from the [netbooks](http://netbooks.networkmedicine.org/)  for more detailed info on how to get them.
